### ISSUE TITLE
Remove deprecated Maintainer command

### ISSUE THREAD

**Issue by SuperSandro2000** [dtandersen/docker_factorio_server#215]
The Maintainer tag is deprecated and replaced by a label. Source: https://docs.docker.com/engine/reference/builder/#maintainer-deprecated

Additionally to that Microbadger recommends the following tags
````
      org.label-schema.build-date=$BUILD_DATE \
      org.label-schema.name="Kibitzr" \
      org.label-schema.description="Get notified when important things happen" \
      org.label-schema.url="https://kibitzr.github.io/" \
      org.label-schema.vcs-ref=$VCS_REF \
      org.label-schema.vcs-url="https://github.com/SuperSandro2000/docker-images" \
      org.label-schema.vendor="SuperSandro2000" \
      org.label-schema.version=$VERSION \
      org.label-schema.schema-version="1.0"
`````

I would do a PR if you are open to the change or have any other wishes. Just let me know. 😊

**Comment by Fank** [comment]
Sure you can create a PR if you want to.
Maybe split them in replace the deprecated stuff and add the recommended stuff.

**Comment by SuperSandro2000** [comment]
Closed by #241
